components/details/MetadataTags.tsx

```diff
@@ -0,0 +1,67 @@
+import { memo } from "react";
```

Comment:
I think it would be better to call this component `Head`, as it is the head component of the page.